modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,234 @@
+// Module included in the following assemblies:
```

Comment:
qe doc verification for image-precaching is on-hold until blocker bzs resolved and testing is completed.